release.yml

```diff
@@ -0,0 +1,89 @@
+version: 2.3.2
+
+# JIRA Project where the release issue should be created
+issue:
+  project: ENTSBT
+
+
+# If already created, provide the issue number of the Release, otherwise keep it empty.
+# It will be used by the command create-component to link the stakeholder issues
+#key:         # SB-1484
+
+schedule:
+  # Release date scheduled. It will also be used to populate the description to be created for each Jira stakeholder: component or starter owner
+  release: 2020-09-14
+  # Due date for the component/starter owner to release their new components, QE signoff it
+  due: 2020-08-18
+  # EOL of the Snowdrop release
+  eol: 2021-09-30
+
+components:
+  # List of JIRA Projects contributing to a Snowdrop release
+  # They will be used to create a component or starter
+  - name: Hibernate / Hibernate Validator / Undertow
+    jira: EAPSUP
+    properties:
+      - hibernate
+      - hibernate-validator
+      - undertow
+
+  - name: Tomcat
+    jira: JWS
+    properties:
+      - tomcat
+
+  - name: Keycloak
+    jira: KEYCLOAK
+    properties:
+      - keycloak
+
+  - name: AMQP
+    jira: ENTMQCL
+    properties:
+      - amqp-10-starter
+
+  - name: RESTEasy
+    jira: RESTEASY
+    properties:
+      - resteasy
```

Comment:
I might be forgetting something but why do we need to specify a RESTEasy version when we already specify a RESTEasy starter? Wouldn't the starter bring all that is needed?

Reply:
This question is not meant to block this PR. It is rather a more generic question about the dependency included in the BOM.

Reply:
Theoretically, we don't need them but I suspect that we needed them to override the RESTEasy version from the starter at some point. Don't recall for sure. See: 40f61485bbcead43b7ed781e1fa885b0db7b6433